### ISSUE TITLE
Feature Request: Also truncate the positions of all the blizz windows when using an aspect ratio more narrow than the screen. 

### ISSUE THREAD

**Issue by Whiskeypits** [GoldpawsStuff/AzeriteUI#68]
Not really sure what is causing it, but 12/17/20 and newer updates remove the UI and can't find a way to get them back. They just default to the regular BlizzardUI. However, 12/16/20 works just fine. Not really sure what is causing it, but it is pretty strange.

**Comment by goldpaw** [comment]
Try not using MoveAnything.

**Comment by Whiskeypits** [comment]
This fixed it, but will there be a fix to be more compatible with moveanything soon?

**Comment by bsides** [comment]
> This fixed it, but will there be a fix to be more compatible with moveanything soon?

Had to come here for this, a real bummer =/

**Comment by goldpaw** [comment]
It's not a bug, it's a feature. MoveAnything moves things not supposed to be moved, and in nearly all cases causes other bugs. I don't want the UI to be used along with it. Sorry.

**Comment by bsides** [comment]
I agree @goldpaw, it's just not obvious what causes it when you join the game. MoveAnything is a very very old addon and works as is for a very long time. Wouldn't it be better to show the user a popup with this info maybe? Or put this info into a FAQ?

**Comment by goldpaw** [comment]
Or in an intro message when logging on. Like I'm fairly certain I've done. At least in my tests. -_-

**Comment by Whiskeypits** [comment]
I have a request then. I have an ultrawide monitor. I also stream with your UI. Could you possibly move the windows such as character, map, talents, ect. along with the UI when switching to 16:9 ratio? Basically when I switch to 16:9 ratio while using my ultrawide, the windows still adhear to 21:9. Moving them with the aspect ratio would would fix all problems. Thats the reason why I used moveanything in the first place. If it is too much I understand. Thanks for responding.

**Comment by goldpaw** [comment]
Done! 
Just go to our cogwheel menu, click "aspect ratio", and select "16:9"!
:smile:

**Comment by goldpaw** [comment]
Wait, no I misread that, sorry! You meant the blizzard elements! 

Well, that would be... a job. But might be doable. I'll see if there is any semi-intelligent way I can make it happen.

**Comment by Whiskeypits** [comment]
Thank you so much. That would be an amazing feature for sure!

**Comment by goldpaw** [comment]
Changed the title of this, so others can see this is now something I'm working on. :)
